rules/windows/process_creation/proc_creation_win_susp_rundll32_script_run.yml

```
@@ -4,25 +4,28 @@ status: experimental
 description: Detects suspicious process related to rundll32 based on arguments
 references:
     - https://gist.github.com/ryhanson/227229866af52e2d963cf941af135a52
-    - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1218.011/T1218.011.md
-author: frack113
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1218.011/T1218.011.md
```

```suggestion
    - https://github.com/redcanaryco/atomic-red-team/blob/cd3690b100a495885c407282d0c94c85f48a8a2e/atomics/T1218.011/T1218.011.md
```
I think that's what @frack113 meant. It has it's up- and downsides but in the end I also think we benefit from being able to understand what the rule author was looking at.